crates/bevy_tasks/src/task_pool.rs

```diff
@@ -230,23 +292,52 @@ impl TaskPool {
                         break result;
                     };
 
-                    self.executor.try_tick();
+                    self.compute_executor.try_tick();
```

Comment:
isn't ticking all three executors here going to run them all with the same priority when the scope is active?

Reply:
Yes, mostly because we cannot tell which group initiated the scope. We could try storing the information in a the thread-local, but I'm not sure if that's the best idea.

Reply:
Changed it so that both `scope` and `spawn` specify which `TaskGroup` a task is to be scheduled onto, which makes this a non-issue.